index.html

```diff
@@ -1,12 +1,101 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+
+    <link rel="stylesheet" href="https://rapidui.netlify.app/styles/main.css" />
+
+    <link rel="stylesheet" href="./index.css" />
+    <link rel="stylesheet" href="./components/homepage/homepage.css" />
+
     <title>Rapid Quiz</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <div class="page-layout">
+      <nav class="quiz-navbar">
+        <div class="logo">
+          <a href="#" class="quiz-navbar-text">
+            <span class="font-family-kaushan">R</span>apid
+            <span class="font-family-kaushan">Q</span>uiz</a
+          >
+        </div>
+      </nav>
+
+      <!-- --- main body ---- -->
+      <main class="quiz-main-container">
+        <h2>
+          Unlocking knowledge at
+          <span class="font-family-kaushan">RAPID</span> speed !
+        </h2>
+        <p class="category-heading">Choose your quiz category</p>
+
+        <div class="category-container">
+          <a href="./components/rules/rules.html">
+            <div class="card-vertical">
+              <img src="./assets/img/cosmos.jpg" alt="card" />
+              <div class="card-body">
+                <h3 class="card-header">Cosmos</h3>
+                <p class="card-desc">
+                  Want to explore the world of cosmos or to test your knowledge.
+                  Then this is for you !
+                </p>
+              </div>
+            </div>
+          </a>
+          <a href="./components/rules/rules.html">
+            <div class="card-vertical">
+              <img src="./assets/img/tech.jpg" alt="card" />
```

Comment:
please use valid alt text for  better accessibility

Reply:
Done